server/services/ProfilesService.js

```diff
@@ -387,27 +375,133 @@ const unblockProfile = function (myId, blockedProfileId) {
   })
 }
 
-const getFollowingsById = function (profileId) {
-  const profileRef = db.collection('users').doc(profileId)
-
-  return profileRef.get().then(doc => {
-    if (!doc.exists) {
-      throw new CustomError({
-        name: 'DatabaseError',
-        message: 'No profile with given id',
-        status: 404
+const getFollowingsById = function(profileId, followingsLimit, paginationId) {
```

Comment:
getFollowersById and getFollowingsById follow the same structure (their params, chaining and response is generally the same besides probably only the collection name).

Please try rewriting this into single getter based on type with adding one more param aka 'path' ('followings' or 'followers' in our case but could be used even for other types) - believe me after hating me for a few minutes and then properly rewriting it - you will like it yourself

Reply:
> getFollowersById and getFollowingsById follow the same structure (their params, chaining and response is generally the same besides probably only the collection name).
> 
> Please try rewriting this into single getter based on type with adding one more param aka 'path' ('followings' or 'followers' in our case but could be used even for other types) - believe me after hating me for a few minutes and then properly rewriting it - you will like it yourself

Done in pull request- Refactor profile backend #71